docs/store-operations/pricing/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,94 @@
+# Price Lists - International Enhancements for Multi-Storefront
+
+<Callout type="info">
+  Tax-inclusive or exclusive price lists are in beta. Features may change based on user feedback or internal testing.
+</Callout>
+
+_International Enhancements for Multi-Storefront_ allow you to deliver localized experiences to fit different shopper needs. 
+
+Shoppers expect storefront prices to include or exclude tax depending on their region when you sell internationally. 
+
+When you create separate storefront channels for each region, you may need some storefronts to display prices that include tax, while others display prices that exclude tax.
+
+To do so, specify whether a price list has prices that include or exclude tax when entered. Then assign the price list to storefront channels.
+
+## Limitations
+
+- Stores using [Catalog V2](/docs/store-operations/catalog/migration) have fixed pricing and will use global store settings. 
+- You cannot enter price changes due to product modifier rules, gift wrapping features, and fixed shipping prices into a price list. These inherit global settings and always apply on top of the price list.
```

Comment:
Not sure if this sentence is accurate

Reply:
@bc-vinbase is correct.  These will inherit the setting from the price list and will not be treated as the global setting.

Maybe `These inherit global settings and always apply on top of the price list.` should be changed to `These prices will be applied as-is with with the prices entered with tax setting for the current shopper applied` or similar?

Reply:
@megdesko Thanks, clarified it and added an example. I assume you meant to use the $10 modifier rule in your final calculation (to total $75.95)